### ISSUE TITLE
[Suggestion] Would be nice to be able see player names in /axh vault

### ISSUE THREAD

**Issue by VaclavKlima** [Mangonels/AntiXrayHeuristicsIssues#31]
I would like to see in /axh vault player name not just "Handled xrayer slot #number"...
I tried to add {PlayerName} placeholder but did not work..

![image](https://user-images.githubusercontent.com/61498435/131243970-af7e6f5f-b34a-49dc-88dd-ee7336603578.png)

**Comment by Mangonels** [comment]
Yeah sorry, the UI is "kind of" a mess right now. But at least it's functional, and for now you can see the player inside the vault.

**Comment by VaclavKlima** [comment]
Ye but when I want to find a specific player I have to click through all players to find him :D.. or maybe add command to open vault of specific player.. something like /axh vault playername and it will open their vault
